frontend/src/app/components/television/television.component.ts

```diff
@@ -21,13 +22,30 @@ export class TelevisionComponent implements OnInit {
     private apiService: ApiService,
     private stateService: StateService,
     private seoService: SeoService,
+    private route: ActivatedRoute
   ) { }
 
   ngOnInit() {
     this.seoService.setTitle($localize`:@@46ce8155c9ab953edeec97e8950b5a21e67d7c4e:TV view`);
     this.websocketService.want(['blocks', 'live-2h-chart', 'mempool-blocks']);
 
-    this.apiService.list2HStatistics$()
+    this.route.fragment
+      .pipe(
+        switchMap(() => {
```

Comment:
Would be cleaner to use the fragment value in the method `.switchMap((fragment)` than calling `this.route` again. Check `statistics.component.ts`

Reply:
Thanks. Fixed 👍🏻